redirect-rules/simple/README.md

```diff
@@ -0,0 +1,74 @@
+# Redirect Rule: Simple Example with a Webtask
+
+This sample demonstrates a minimal implementation of the `redirect` protocol. It contains an Auth0 [rule](http://auth0.com/docs/rules) that will redirect the user to a simple consent form. If the user checks the "I agree" checkbox on this form and clicks the **Submit** button, they are then redirected back to Auth0 to complete the authentication flow. On future logins they will no longer be prompted since the consent action is stored in their user profile.
+
+The consent form is hosted using a simple [Webtask](https://webtask.io/) that you can easily modify and provision and use in your version of the rule.
+
+## Rule Setup
+
+To try this rule out with your own Auth0 account using an existing instance of the consent form webtask, follow these steps:
+
+1. Create a new Auth0 rule using the contents of the [`rule.js`](rule.js) script.
+1. In your account's [Rules](https://manage.auth0.com/#/rules) screen, add a setting that has the key `CONSENT_FORM_URL` and the following value:  
+  ```
+  https://sandbox.it.auth0.com/api/run/simple-redirect-protocol/consent?webtask_no_cache=1
```

Comment:
I wonder if the minimalistic example should still show creation of a custom redirect URL. After all, the key idea is that people have full control over the UI and data contract between the URL being redirected to and the the code of their rule. To that end, it would be useful to show some first class data passed from the consent form back into the rule code (e.g. date of birth or something).